.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @irajdeep @priyolahiri @mircea-cosbuc @igor-karpukhin @lsierant @slaskawi 
```

Comment:
Not sure why https://github.com/igor-karpukhin is showing up as non-existent, but he's in the team.

Reply:
Probably because of write permissions, safe to ignore my comment.